tsl/test/sql/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(TEST_FILES
   bgw_custom.sql
   bgw_policy.sql
   compression_bgw.sql
+  compression_alter.sql
```

Comment:
Do we need a completely new test for this? We already have ~8 compression test files, including a `compression_ddl`. Every unique test file that we add will add to the test runtime. Maybe it can even be a shared test?

Reply:
There will be more tests specific to alter table. I prefer to split the test files up. It is hard to figure out what is being tested with huge test files.
Moving it to a shared test might work.

Reply:
This is now called from compression_ddl.sql so that we avoid the cost of a new db setup. This file moved to include directory.